lib/MusicBrainz/Server/ExternalUtils.pm

```diff
@@ -9,7 +9,7 @@ our @EXPORT_OK = qw(
 sub get_chunked_with_retry {
     my ($ua, $url) = @_;
     my $response;
-    my $retries_remaining = int(25.0 / $ua->timeout);
+    my $retries_remaining = int(25.0 / $ua->inactivity_timeout);
```

Comment:
This makes it perform no search, since the default LWP timeout is 180 seconds and `int(25.0 / 180)` is 0.

I don't understand what the previous code is calculating, anyway. A fixed number of retries is probably fine?

Reply:
25 seconds was set in https://github.com/metabrainz/musicbrainz-server/commit/7c2299732476f900acb12f5dc4d548bd362ec3ca as an arbitrary period of time. It is no longer needed with the new global timeout.

Retry at most 5 times instead with https://github.com/metabrainz/musicbrainz-server/pull/1102/commits/61fd923c833d27418594395f598cbc96beff9b8c.